ote/src/cljs/ote/ui/form_fields.cljs

```diff
@@ -579,3 +608,110 @@
+                                                       (tr [:csv (get-in data [:csv-count :error])])]
+                         :else [:span]))]
+          :csv-file [:div
+                     [:div.row (stylefy/use-style style-base/divider)]
+                     [:div.row
+                      (csv-help-text)
+                      [:div.row {:style {:padding-top "20px"}}
+                       [field {:name      ::t-service/csv-file
+                               :type      :file
+                               :label     "Tiedosto"
+                               :accept    ".csv"
+                               :on-change on-file-selected
+                               }]]
+                      [:div.row
+                       (let [success (if (get data :csv-imported)
+                                       true
+                                       false)
+                             amount (if (get data ::t-service/companies)
+                                      (count (get data ::t-service/companies))
+                                      nil)]
+                         (when success
+                           [:span {:style {:color "green"}} (tr [:csv :parsing-success] {:count amount})]))]]]
+          :form [:div.row
+                 [:div.row (stylefy/use-style style-base/divider)]
+                 [:div.row
+                  [field {:name                ::t-service/companies
+                          :type                :table
+                          :update!             #(update! {::t-service/companies %})
+                          :table-wrapper-style {:max-height "300px" :overflow "scroll"}
+                          :prepare-for-save    values/without-empty-rows
+                          :table-fields        [{:name      ::t-service/name
+                                                 :type      :string
+                                                 :label     (tr [:field-labels :transport-service-common ::t-service/company-name])
+                                                 :required? true
+                                                 }
+                                                {:name      ::t-service/business-id
+                                                 :type      :string
+                                                 :label     (tr [:field-labels :transport-service-common ::t-service/business-id])
+                                                 :validate  [[:business-id]]
+                                                 :required? true
+                                                 :regex     #"\d{0,7}(-\d?)?"}]
+                          :delete?             true
+                          :add-label           (tr [:buttons :add-new-company])
+                          :error-data          (::t-service/companies (:ote.ui.form/warnings data))}
+                   companies]]]
+          ""  ;; default
+          ))]]))
```

Comment:
ending parens

Reply:
I wanted to leave that "default" commented text and it cleared when these are in different rows